.github/workflows/docker.yml

```
@@ -22,20 +17,41 @@ jobs:
     permissions: {}
     outputs:
       TAG: ${{ steps.determine.outputs.tag }}
+    if: |
+      github.event_name == 'workflow_dispatch'
+      || (
```

Should this be an &&? If event_name is 'workflow_dispatch' and it's a success.. etc.
```suggestion
      && (
```

---

Gotcha, thanks!